staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -14,6 +14,7 @@ import (
 	"k8s.io/klog/v2"
 
 	dbInflux "github.com/kubeedge/Template/data/dbmethod/influxdb2"
+	"github.com/kubeedge/Template/data/dbmethod/mysql"
```

Comment:
How about naming it dbMysql like the other dependencies? Otherwise, the datahandlers may affect each other when adding to the database later.

Reply:
dbMsql